docs/datagrid/performance.md

```diff
@@ -0,0 +1,57 @@
+# Performance and rendering
+Row virtualization is a performance optimization technique used in DataGrid.  Instead of rendering all the rows of data at once, which can be thousands of entries, row virtualization renders only the rows that are currently visible within the user's viewport (plus a small buffer). As the user scrolls, the DataGrid dynamically renders new rows and removes those that scroll out of view.
```

Comment:
Personally I don't find anything wrong, but Grammarly suggests to leave out _that are_.